src/main/resources/mcmod.info

```diff
@@ -11,6 +11,6 @@
       "credits": "",
       "logoFile": "assets/refinedstorage/textures/logo.png",
       "screenshots": [],
-      "dependencies": []
+      "dependencies": ["inventorytweaks"]
```

Comment:
"List of optional mods ("soft" dependencies) that should load before this mod." IT is just a setting to make sure it loads before RS if it is present so there's no issues calling classes on it before it's loaded. "requiredMods" is the setting that lists hard dependencies that are always needed.